integration-tests/deploy_R_models.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import, print_function
```

Comment:
Thanks @Corey-Zumar for the review.
I tried to code it in similar way as that of <https://github.com/ucbrise/clipper/blob/develop/integration-tests/deploy_PySpark_models.py>.
However, if I have to turn it into a Python unit test, what tests other than whatever present present in this file should I include?